database/seeders/DatabaseSeeder.php

```diff
@@ -18,7 +18,7 @@ public function run() {
         $this->call(FollowTableSeeder::class);
         TrainStation::factory()->count(50)->create();
         $this->call(EventSeeder::class);
-        HafasTrip::factory()->count(50)->create();
+        $this->call(HafasTripSeeder::class);
```

Comment:
Whats the point of this if you just put the old line in an extra file with 18 lines?

Reply:
If a Factory doesn't have a Seeder, you can't execute this exact line from the CLI.

For testing purposes I had to execute 
```
> php artisan db:seed --class=HafasTripSeeder     
```
and afterwards
```sh
> php artisan db:seed --class=TrainCheckinSeeder     
```